python/cudf/cudf/tests/test_csv.py

```diff
@@ -2276,3 +2276,9 @@ def test_read_compressed_BOM(tmpdir):
         f.write(buffer)
 
     assert_eq(pd.read_csv(fname), cudf.read_csv(fname))
+
+
+@pytest.mark.parametrize("buffer", ["1", '"one"'])
```

Comment:
Question: do we need to include a cpp test as well with this input?

Reply:
I don't think so. both cudf and libcudf end up calling the same (buggy) code in the same way.